### ISSUE TITLE
Merge Client & Server

### ISSUE THREAD

**Issue by phase** [phase/PigeonLauncher#3]
1. [x] Make mappings from Server to Client
2. [x] Inject server only classes into client jar
3. [ ] Decompile & Diff shared classes, observe changes

**Comment by phase** [comment]
Here's a (really great) drawing of what's happening during build time.

![shitty drawing](https://camo.githubusercontent.com/edd72d50696c40dec2b1b53cf178eeec23a30017/68747470733a2f2f692e696d6775722e636f6d2f4c425a4e7046432e706e67)

The only thing left is to modify the shared classes to support the server only classes.
This is currently waiting on SpongePowered/Mixin#247.

**Comment by phase** [comment]
This issue has been superseded by [BetaGradle](https://github.com/PigeonMC/BetaGradle), which will create the full jar for the launcher to use.
